packages/server/src/AccountsServer.js

```diff
@@ -242,8 +242,27 @@ export class AccountsServer {
   removeEmail(userId: string, email: string): Promise<void> {
     return this.db.removeEmail(userId, email);
   }
-  verifyEmail(userId: string, email: string): Promise<void> {
-    return this.db.verifyEmail(userId, email);
+  async verifyEmail(token: string): Promise<void> {
+    const user = await this.db.findUserByEmailVerificationToken();
+    if (!user) {
+      throw new AccountsError({
+        message: 'Verify email link expired',
+      });
+    }
+    const tokenRecord = find(user.services.email.verificationTokens,
```

Comment:
@TimMikeladze will all dbInterfaces return the user object formatted in the same way ?